<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tests</class>
 <widget class="QMainWindow" name="tests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>862</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>950</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>245</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>117</green>
        <blue>113</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>161</red>
        <green>157</green>
        <blue>151</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>245</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>245</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>117</green>
        <blue>113</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>161</red>
        <green>157</green>
        <blue>151</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>245</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>245</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>117</green>
        <blue>113</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>161</red>
        <green>157</green>
        <blue>151</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>241</red>
        <green>235</green>
        <blue>226</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="GenericDisplayButton" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background: rgb(49, 54, 59);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_turnoncenter">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_turnoffcenter">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setvalue">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SetValue</string>
    </property>
   </widget>
   <widget class="PoleLevelIndicator" name="widget_pole" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="GenericDisplayButton" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid red;


</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enable">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disable">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_duration">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextstep">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>NextStep</string>
    </property>
   </widget>
   <widget class="ClickableGeometricShape" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid red;
</string>
    </property>
   </widget>
   <widget class="HorisontalLevelBars" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid red;
</string>
    </property>
   </widget>
   <widget class="Histogram" name="widget_histogram" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>271</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_key">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_value">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>490</y>
      <width>65</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>550</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="FancyDisplay" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionOn">
   <property name="text">
    <string>ON   </string>
   </property>
  </action>
  <action name="actionOff">
   <property name="text">
    <string>OFF  </string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGrille_ON">
   <property name="text">
    <string>Grille ON</string>
   </property>
  </action>
  <action name="actionGraphes">
   <property name="text">
    <string>Graphes</string>
   </property>
  </action>
  <action name="grilleON">
   <property name="text">
    <string>Grille ON</string>
   </property>
  </action>
  <action name="grilleOFF">
   <property name="text">
    <string>Grille OFF</string>
   </property>
  </action>
  <action name="incGraphe">
   <property name="text">
    <string>Graphe +</string>
   </property>
  </action>
  <action name="actuGraphe">
   <property name="text">
    <string>Graphe</string>
   </property>
  </action>
  <action name="decGraphe">
   <property name="text">
    <string>Graphe -</string>
   </property>
  </action>
  <action name="Test_Sommet_ON">
   <property name="text">
    <string>Test Sommet ON</string>
   </property>
  </action>
  <action name="Test_Sommet_OFF">
   <property name="text">
    <string>Test Sommet OFF</string>
   </property>
  </action>
  <action name="actionCrb_ON">
   <property name="text">
    <string>Crb ON</string>
   </property>
  </action>
  <action name="actionCrb_OFF">
   <property name="text">
    <string>Crb OFF</string>
   </property>
  </action>
  <action name="actionSave_Graphe">
   <property name="text">
    <string>save Graphe</string>
   </property>
  </action>
  <action name="actionLoad_Graphe">
   <property name="text">
    <string>load Graphe</string>
   </property>
  </action>
  <action name="actionClear_Graphe">
   <property name="text">
    <string>clear Graphe</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericDisplayButton</class>
   <extends>QWidget</extends>
   <header>GenericDisplayButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PoleLevelIndicator</class>
   <extends>QWidget</extends>
   <header>PoleLevelIndicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClickableGeometricShape</class>
   <extends>QWidget</extends>
   <header>ClickableGeometricShape.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HorisontalLevelBars</class>
   <extends>QWidget</extends>
   <header>HorisontalLevelBars.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Histogram</class>
   <extends>QWidget</extends>
   <header>Histogram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FancyDisplay</class>
   <extends>QWidget</extends>
   <header>FancyDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
